test/e2e/scenarios/ACL/acl_update_with_tags.yaml

```diff
@@ -60,7 +60,7 @@ steps:
         conditions:
           ACK.ResourceSynced:
             status: "True"
-            timeout: 100
+            timeout: 10
```

Comment:
Hmmm. Not related to this PRs changes, but this test only tests half of the output. To test the controllers we not only want to test that the K8s resource reached `ACK.ResourceSynced` but we also want to describe the AWS resource and make sure the tags appear on there too. We aren't doing any validation in any of these tests?

Reply:
Should we add expect there? To be honestly I don't think our test framework supports Expect compare for Spec. Also, it added unit test for custom compare tags functions. Could the unit test handle this case?

Reply:
Will fix that issue by a new PR to change test framework